contracts/feature-tests/async/forwarder-raw/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+#![no_std]
+
+elrond_wasm::imports!();
+
+/// Test contract for investigating async calls.
```

Comment:
This contract is clearly for testing, but there are no mandos tests that reference it.

Reply:
Yeah, coming this weekend

Reply:
Perhaps you forgot to stage them before commiting?